@types/TokenBuildInputArray.d.ts

```diff
@@ -0,0 +1,10 @@
+type TokenBuildInputItem = {
```

Comment:
is @types folder new? If so, can we move this to a non-namespaced folder to avoid confusion with @types npm package namespace?

Reply:
You mean using `types` ? I thought this would be beneficial in @types as a consumer would maybe like to use the type when they define their own inputs?

But of course at the moment there is no way to get to them as they are internal to the function.

Reply:
Sounds good. I was more referring to calling the folder `@types`, which is a little confusing to me because that's usually for local overrides or stubbing of node_modules. E.g from PRC: https://github.com/primer/react/tree/main/%40types

I was suggesting moving this to something like `/src/types/build-helpers`.. and then reexport from an index.d.ts if you want consumers to use it through the npm package.